pkg/manager/task_autotag.go

```diff
@@ -586,6 +595,7 @@ func (t *autoTagFilesTask) processGalleries(r models.ReaderRepository) error {
 }
 
 func (t *autoTagFilesTask) process() {
+	begin := time.Now()
```

Comment:
should this and the relevant log print be run in 
`func (j *autoTagJob) Execute(ctx context.Context, progress *job.Progress) {`  instead ?
When doing a specific autotask no time stats are printed.